packages/synthetix-main/contracts/storage/AccountStorage.sol

```diff
@@ -0,0 +1,19 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/satellite/SatelliteFactory.sol";
+
+contract AccountStorage {
+    struct AccountStore {
+        bool initialized;
+        SatelliteFactory.Satellite account;
+        mapping(uint256 => mapping(address => uint16)) accountDelegations;
```

Comment:
This mapping contains the account delegations. Each account can be delegated to more than one address and each address can have one or several atomic permissions (mapped into a uint16, 1 bit per permission). 

I'm tempted to up the limit from uint16 to uint64 or something like that to have room for more permissions, but up to now only 7 are defined.